packages/openapi-to-graphql/src/types/options.ts

```diff
@@ -132,7 +145,9 @@ export type Options = {
    * non-standard authentication requirements.
    */
   customResolvers?: {
-    [title: string]: { [path: string]: { [method: string]: ResolveFunction } }
+    [title: string]: {
+      [path: string]: { [method: string]: ResolveFunction | ResolveObject }
+    }
   }
```

Comment:
As the Subscription field wait for a required `subscribe` field and optional `resolve`, the `customResolvers` had to be adapted.